graphql_compiler/__init__.py

```diff
@@ -3,6 +3,7 @@
 from .compiler import CompilationResult, OutputMetadata  # noqa
 from .compiler import compile_graphql_to_gremlin, compile_graphql_to_match  # noqa
 from .query_formatting import insert_arguments_into_query  # noqa
+from .query_formatting.graphql_formatting import pretty_print_graphql  # noqa
 
```

Comment:
if you want to save yourself a file, you could `import .query_formatting.graphql_formatting as tool` here, and move the logic from `tool.py` into `__main__` there.  A little nasty, but does save a file!